arbeitszeit_web/www/presenters/change_user_email_address_presenter.py

```diff
@@ -0,0 +1,34 @@
+from dataclasses import dataclass
+
+from arbeitszeit.use_cases import change_user_email_address
+from arbeitszeit_web.notification import Notifier
+from arbeitszeit_web.translator import Translator
+from arbeitszeit_web.url_index import UrlIndex
+
+
+@dataclass
+class ViewModel:
+    redirect_url: str | None
```

Comment:
Lexi brings up a good point here.  What is the rationale for using the union type str | None rather than simply str and having the empty string mean "no URL"?

Reply:
As discussed yesterday I will merge this as-is. We can keep this in mind and discuss later if desired.